### PR TITLE
Add class when ad is fullscreen

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Would be useful for styling

### CODE REVIEW

File: src/videojs.ima.js [hunk 1]
```diff
@@ -428,6 +428,7 @@
         fullscreenDiv.className = 'ima-fullscreen';
         adContainerDiv.style.width = window.screen.width + 'px';
         adContainerDiv.style.height = window.screen.height + 'px';
+        adContainerDiv.className = 'ima-ad-container--fullscreen';
```
Comment (tensafefrogs) [inline src/videojs.ima.js]: Why the double dashes here? Would one be ok: "ima-ad-container-fullscreen" ?

Comment (ironsidevsquincy) [inline src/videojs.ima.js]: Sorry for my tardiness, was using a bem style class naming, but as it's not being used elsewhere in the codebase, sure, single dash it is

Comment (ironsidevsquincy) [inline src/videojs.ima.js]: actually, no longer have access to the guardian's repo. maybe @sndrs wouldn't mind changing it...

**Comment by shawnbuso** [comment]
Are you able to update the existing pull request branch? If not feel free to open another and we can close this.

**Comment by shawnbuso** [comment]
Closing for lack of activity - if you're able to update and want to push feel free to re-open.

**Pull request closed without merging**